CHANGES

```diff
@@ -1,8 +1,20 @@
-2.5.1: (trunk)
+2.5.0:
+* Remove `Mirage.vchan`, `Mirage.vchan_localhost`, `Mirage.vchan_xen` and
+  `Mirage.vchan_default`. Vchan users need to adapt their code to directly
+  use the `Vchan` functors instead of relying on the combinators.
+* Remove `Mirage.conduit_client` and `Mirage.conduit_server` types.
+* Change the type of the `Mirage.conduit_direct` combinator.
+  Previously, it took an optional `vchan` implementation, an optional
+  `tls` immplementation and an optional `stackv4` implemenation. Now,
+  it simply takes a `stackv4` implementation and a boolean to enable
+  or disable the `tls` stack. Users who want to continue to use
+  `vchan` with `conduit` should now use the `Vchan` functors inside
+  `unikernel.ml` instead of the combinators in `config.ml`.
+* Change the type of the `Mirage.http_server` combinator. The first argument
+  (the conduit server configuration) is removed and should now be provided
```

Comment:
Could you move the http server one to the top; it affects almost everyone. An example of before after would be useful to make it really easy